modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -333,7 +333,8 @@ public void processOpts() {
                 .put("lowercase", new LowercaseLambda().generator(this))
                 .put("uppercase", new UppercaseLambda())
                 .put("titlecase", new TitlecaseLambda())
-                .put("camelcase", new CamelCaseLambda().generator(this))
```

Comment:
Is this a change from master?

Reply:
No, I added this to allow the default `camelize`behaviour, i.e. uppercasing the first letter, to be available as lambda in the templates.